pgn_downloader/main.py

```diff
@@ -8,17 +8,29 @@ def cli_arguments(args=None):
     """Parse CLI arguments"""
     parser = argparse.ArgumentParser()
     parser.add_argument("username")
+    parser.add_argument("color", help="Color (white/black)")
```

Comment:
This could use the `choices` option of argparse to force the value to white / black
https://docs.python.org/3/library/argparse.html#choices

Reply:
Also this should be optional, right? So that you can still download all games if needed

Reply:
Yeah, I change it to being optional, such that the default is downloading all games and if the color argument is specified, then only the games of one color